config/simplesamlphp-users.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/*
+  Specify user info for https://hub.docker.com/r/kristophjunge/test-saml-idp/ in the same format
+  as our Seneca IdP will give it.  This is modeled on the original config here:
+  https://github.com/kristophjunge/docker-test-saml-idp/blob/master/config/simplesamlphp/authsources.php
+*/
+$config = array(
+
+    'admin' => array(
+        'core:AdminPassword',
+    ),
+
+    'example-userpass' => array(
+        'exampleauth:UserPass',
+        'user1:user1pass' => array(
+            'uid' => array('1'),
+            'eduPersonAffiliation' => array('group1'),
+            /*
+              NOTE: we add a bunch of claims that we expect to get back from Seneca,
+              and need to simulate here.
+            */
+            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress' => 'user1@example.com',
```

Comment:
Change the email address to match the `'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name'` claim (same email in all claims).  Do this for every account.